src/Query/QueryBuilder.php

```diff
@@ -1870,32 +1881,9 @@ protected function extractAlias(string $table)
         return false;
     }
 
-    public function buildWith($withs, &$params): string
```

Comment:
Why is it removed?

Reply:
It was replaced by buildWithQueries (), both methods are the same, and you can check that it is not used.